README.md

```diff
@@ -899,6 +900,66 @@ and need to be removed by hand.
 For long term use, a dedicated project in the Google Cloud Platform is
 recommended to prevent automated manipulation of important machines.
 
+<a name="openstack"/>
+
+## Openstack backend
+
+The Openstack backend is easy to setup and use, and allows distributing
+your tasks to an openstack environment.
+
+_$PROJECT/spread.yaml_
+```
+(...)
+
+backends:
+    openstack:
+        plan: cpu2-ram4-disk10
+        halt-timeout: 2h
+        systems:
+            - ubuntu-20.04:
+                  image: ubuntu-focal-daily-amd64
+                  workers: 5
+
+            # Extended syntax:
+            - another-system:
+                image: some-other-image
+                networks:
+                    - network_external
+                    - network_pvn
+                groups:
+                    - group_external
+
+```
+
+The Openstack backend gets all the information to authenticate from the
+environment variables. The following variables have to be set:
+```
+OS_AUTH_URL
+OS_PROJECT_ID
+OS_PROJECT_NAME
+OS_USER_DOMAIN_NAME
+OS_PROJECT_DOMAIN_ID (optional)
+OS_TENANT_ID
+OS_TENANT_NAME
+OS_USERNAME
+OS_PASSWORD
+OS_REGION_NAME
+OS_INTERFACE
+OS_IDENTITY_API_VERSION
```

Comment:
In addition to the above, which of those variables have a direct equivalent in the Google backend setup?

Reply:
In google we use the env var SPREAD_GOOGLE_KEY which has a link to the file with the following data:
"type"
"project_id"
"private_key_id"
"private_key"
"client_email"
"client_id"
"auth_uri"
"token_uri"
"auth_provider_x509_cert_url"
"client_x509_cert_url"

I understand that the equivalent are 
OS_PROJECT_ID <-> project_id
OS_AUTH_URL <-> auth_uri